Description: Disable CRL monitor on all architectures
 CRL monitor is unavailable on Debian architecture kFreeBSD, causes FTBFS
Author: Felix Lechner <felix.lechner@lease-up.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=860514
Forwarded: not-needed
Last-Update: 2017-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: wolfssl-5.6.6/configure.ac
===================================================================
--- wolfssl-5.6.6.orig/configure.ac
+++ wolfssl-5.6.6/configure.ac
@@ -778,7 +778,7 @@ then
     test "$enable_hrrcookie" = "" && enable_hrrcookie=yes
     test "$enable_fallback_scsv" = "" && enable_fallback_scsv=yes
     test "$enable_webserver" = "" && enable_webserver=yes
-    test "$enable_crl_monitor" = "" && enable_crl_monitor=yes
+    test "$enable_crl_monitor" = "" && enable_crl_monitor=no
     test "$enable_sni" = "" && enable_sni=yes
     test "$enable_maxfragment" = "" && enable_maxfragment=yes
     test "$enable_alpn" = "" && enable_alpn=yes
